src/main/java/org/folio/rest/impl/ModTenantAPI.java

```diff
@@ -165,6 +193,45 @@ private String getConfigValue(String configName) {
     return configEntryValueField.encode();
   }
 
+  private Future<String> initDatabase(Map<String, String> okapiHeaders, Vertx vertx) {
+    Promise<String> promise = Promise.promise();
+    String tenantId = okapiHeaders.get(OKAPI_TENANT);
+    vertx.executeBlocking(blockingFeature -> {
```

Comment:
Why it should be "executeBlocking"?

Reply:
> Why it should be "executeBlocking"?

Writing this code i've relied on code into SRS and mod-pub-sub and they use executeBLocking, so i've used the same approach.